CHANGELOG.md

```diff
@@ -21,6 +21,7 @@ Given the `MAJOR.MINOR.PATCH` pattern, here is how we decide to increment:
 ### Changed
 
 ### Removed
+- Disables tests for landing page events, since we don't currently have events.
```

Comment:
There are no upcoming events. We could create mock events to get the tests to pass but they'll just break again down the road and we need to have design/content see the no events state anyway for if there should be some text there:

![screen shot 2015-09-09 at 1 49 47 pm](https://cloud.githubusercontent.com/assets/704760/9769691/ad40c8c6-56f9-11e5-90a6-cf5db261da9e.png)

cc @duelj @sarahsimpson09